Example/Tests/Tests.m

```diff
@@ -103,7 +103,28 @@
         it(@"sets groupId", ^{
             SEGGroupPayload *payload = [[SEGGroupPayload alloc] initWithGroupId:@"322" traits:@{} context:@{} integrations:@{}];
             [integration group:payload];
-            [verify(amplitude) setGroup:@"[Segment] Group" groupName:@"322"];
+            [verify(amplitude) setGroup:@"322" groupName:@"[Segment] Group"];
+        });
+
+        it(@"settings.groupTypeValue and settings.groupTypeTrait", ^{
+            integration = [[SEGAmplitudeIntegration alloc] initWithSettings:@{ @"groupTypeValue" : @"industry",
+                                                                               @"groupTypeTrait" : @"company" }
+                                                               andAmplitude:amplitude
+                                                              andAmpRevenue:amprevenue];
+            SEGGroupPayload *payload = [[SEGGroupPayload alloc] initWithGroupId:@"32423084" traits:@{
+                @"company" : @"Segment",
+                @"industry" : @"Technology"
```

Comment:
This test looks ok, but I think it's a bit confusing since groupType:groupTrait should follow the pattern general:specific. Here, looks like specific:general (i.e. Segment:Technology).

Reply:
makes sense, will change